server/models/alumniModel.ts

```diff
@@ -0,0 +1,31 @@
+import mongoose, { Schema } from "mongoose";
+import { IAlumni } from "../types/alumni";
+
+const alumniSchema = new Schema<IAlumni>(
+  {
+    company: { type: String, trim: true },
+    name: { type: String, trim: true, required: true },
+    email: { type: String, trim: true, required: true },
+    linkedIn: { type: String, trim: true },
+    campus: { type: String, trim: true },
```

Comment:
could maybe set this up as an enum depending on how we parse the alumni sheet we're working with

Reply:
Like use enums for some of the fields? Like campus? Maybe Im misunderstanding. Lets chat!